platform/default/src/mbgl/storage/online_file_source.cpp

```diff
@@ -466,8 +468,8 @@ void OnlineFileRequest::networkIsReachableAgain() {
 }
 
 void OnlineFileRequest::setTransformedURL(const std::string&& url) {
```

Comment:
here it is better to change to `setTransformedURL(std::string&& url)`, or simply to `setTransformedURL(std::string url)`